itest/burn_test.go

```diff
@@ -301,3 +302,112 @@ func testBurnAssets(t *harnessTest) {
 	)
 	AssertBalanceByID(t.t, t.tapd, simpleGroupCollectGen.AssetId, 0)
 }
+
+// testBurnGroupedAssets tests that some amount of an asset from an asset group
+// can be burnt successfully.
+func testBurnGroupedAssets(t *harnessTest) {
```

Comment:
So the main difference from this test to the existing one is that we burn assets from a second trance instead of the first one of a group?
https://github.com/lightninglabs/taproot-assets/blob/9943ff1390dca7965defc98cd0062e673b23e509/itest/burn_test.go#L257

Reply:
Yes, that's all it does.